django/db/migrations/autodetector.py

```diff
@@ -1038,6 +1088,7 @@ def _generate_altered_foo_together(self, operation):
                 )
 
     def generate_altered_unique_together(self):
+        # TODO: Add deprecation warnings here.
```

Comment:
I don't think so -- the autodetector isn't a public API that's meant to be extended.